pkg/webhook/validatingwebhook/validate_ssp_request_test.go

```diff
@@ -0,0 +1,119 @@
+package validatingwebhook
+
+import (
+	"bytes"
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"text/template"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+	"github.com/codeready-toolchain/member-operator/pkg/webhook/validatingwebhook/test"
+
+	userv1 "github.com/openshift/api/user/v1"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func TestHandleValidateSSPAdmissionRequest(t *testing.T) {
```

Comment:
Let's add a test for the case when the request is allowed.

Reply:
Added more tests in 40506aa4d1364d456cb1f690d9f5a86bc97ab362